### ISSUE TITLE
Why is the Identity Profile not included in the list of "standardization work"?

### ISSUE THREAD

**Issue by jtibbetts** [w3c/vc-data-model#25]
In the Introduction there are "two pieces of possible standardization work": 1) data model of entity credentials and 2) specific representations in different syntaxes.  Why isn't the Identity Profile Model (cf section 3.2 of this document) included in that list?  If it's more fully described in some other document or standard, then that should be referenced.  If it's defined herein, it should be added to this list.

**Comment by stevenrowat** [comment]
An interesting observation, and after reading through those sections again my guess (not being an author of this) is a combination of the following:

1.  Perhaps if "the minimum information necessary to create and use entity credentials" is achieved, then the required architecture is also achieved to make Identity Profiles; but not the other way around.
2. There's been reported, on the mailing list, to be a lot of politics around "Identity" that might be best avoided by people who see it as a morass; and since (as someone also said) the VC work will not "solve" Identity on the Internet, perhaps not mentioning it, when the opportunity presented itself, was attractive.

I agree that it looks like a major omission in the flow of the document though, because of the equal space and size given to the Entity Credential and Identity Profile.

So, personally at this point, I can live with it but I'm not completely happy. :-)

**Comment by jandrieu** [comment]
When third party claims combine with the principle of minimal disclosure, there can't be a definitive Identity Profile Model. Some recipients of claims will need certain information, others different information. Some claim creators will present information in one particular way, e.g., "DateOfBirth", while others will issue minified abstract assertions such as "Over21", "Over18", "AtLeast13", etc. 

The set of information stored as an "identity profile" will change over time and the subset presented to any given recipient should be dynamically composable by the presenter at the point & time of presentation.

The "identity substrate" as envisioned in the self-sovereign identity and rebooting web of trust conversations is an emerging, distributed, reliable, set of verifiable information that can be used to correlate individuals across contexts on-demand, depending on need. By design, its schema is emergent.  The trick for Verifiable Claims is to provide an atomic minimal schema that allows claim creators and controllers to represent whatever identity-enabling information is appropriate to their particular use cases.

Previous VCTF language used the term "Identity", defined as the collection of all claims about a person. This mapped to related ISO standards. However, I've been advocating a shift away from defining "Identity" in that way (see http://bit.ly/identitycrisis for the Rebooting Web of Trust paper on this) and to instead describe identity systems in how they enable or prevent correlation. As a result, the VCTF language shifted to use "Identity Profile" to capture the same concept, namely that there is a collection of identity-enabling information available in the form of Verified Claims. Where those claims are stored, how they are retrieved, managed, and presented gets into the realm of protocol, i.e., beyond our scope, but the assumptions around distributed control and availability strongly suggest that a definitive closed-form data model for the "Identity Profile" would be arbitrarily limiting. My experience with related attempts to define a universal schema for identity suggest that it is, in fact, an unachievable goal exactly because of the innate contextual flexibility required by recipients for identity assurance.

**Comment by jandrieu** [comment]
The short answer is that the schema of the Identity Profile is inherently emergent. Different creators and recipients will have different needs and those needs will change over time.

The VCTF previously used to use the term "Identity", defined as all the claims about a particular person, which is aligned with related ISO standards. I advocate against using "Identity" in that way because it is confusing and rarely captures the needed complexity, especially for real-world uses of identity aka "legal identity". Instead, I argue in favor of describing identity systems in terms of how they enable or prevent correlation.  (You can read the Rebooting Web of Trust paper http://bit.ly/identitycrisispaper for a more complete argument.)  

In part in response to that conversation, VCTF turned to "Identity Profile" to capture essentially the same idea: the collection of all the identity-enabling information stored as verifiable claims.  However, the schema for that profile will, by design, be flexible over time as uses cases, technologies, and regulations evolve.

My experience with different attempts to define a universal schema for "identity" suggest that it is an intractable problem due to the continually changing intersection of the need for minimal disclosure and the flexibility required by recipients for identity assurance.  There's still value in referring to the set of identity-enabling claims collectively and "Identity profile" is as good a term as any. There just isn't a closed form data model for what information can actually be captured in said profile.

**Comment by jandrieu** [comment]
I withdraw my earlier comment without prejudice. ;)

The Entity Profile is missing, and in the latest PR #67, incorrect as commented in https://github.com/w3c/vc-data-model/pull/67#issuecomment-320500440 

I answered a more overarching question because I didn't understand that Entity Profiles were what holders present to inspector-verifiers. I spent a year thinking claims were presented, and therefore interpreted Entity Profiles to be the collection of claims related to an individual, which for various reasons is often the end-goal for many new to issues of digital identity, i.e. "What are the necessary attributes to prove someone's identity". *That* question remains intractable. 

However, for any given use, we can and should demonstrate the Entity Profile actually presented to the inspector-verifier.
